docs/concepts/what-is-a-namespace.md

```diff
@@ -13,10 +13,11 @@ A single Namespace is still multi-tenant.
 You can use Namespaces to match the development lifecycle; for example, having separate `dev` and `prod` Namespaces.
 Or you could use them to ensure Workflow Executions between different teams never communicate; such as ensuring that the `teamA` Namespace never impacts the `teamB` Namespace.
 
-- By default a Temporal Cluster has one "default" and one internal Namespace.
-  All APIs and tools, such as the UI and CLI, default to the "default" Namespace if it is not specified.
-  So, if you are not planning to use multiple Namespaces, we recommend using the default one.
-- **Case Insensitive**: Because of the DNS, namespaces are case insensitive on the network/routing part.
+- The Temporal Cluster uses the Namespace, “default”, if no other Namespace is specified, for all Temporal SDKs and the tctl.
+  - If you are deploying through Docker Compose or using the [auto-setup image](https://github.com/temporalio/docker-builds/blob/main/docker/auto-setup.sh) in a custom Docker Compose application, the Namespace, "default", is created, through the auto-setup script.
+  - If you are deploying through the [Temporal Helm charts](https://github.com/temporalio/helm-charts), you can create the "default" Namespace through the tctl, for example: `tctl namespace default`.
+    We recommend using the default Namespace if you aren’t using multiple Namespaces.
+- **Case Insensitive**: Because of the DNS, namespaces are case-insensitive on the network/routing part.
```

Comment:
```suggestion
- **Case Insensitive**: Because of DNS, Namespaces are case insensitive on the network and routing side.
```
I also recommend lowercasing "Insensitive" in the run-in heading. If you do, please also lowercase "Configuration" in the "Namespace Configuration" run-in heading at the bottom of this list.